test/integration/emulator/SfuClient.h

```diff
@@ -37,6 +40,41 @@ class MediaSendJob : public jobmanager::Job
     memory::UniquePacket _packet;
 };
 
+struct RtxStats
+{
+    // Receiver's stats: how much we missed, recovered and complained.
+    size_t rcvPacketsMissing = 0;
+    size_t rcvPacketsRecovered = 0;
+    size_t rcvNackRequestSent = 0; //  Not implemented: SfuClient does not sent NACK yet.
```

Comment:
received sent ?

Reply:
when we have a group of metrics like this it is better to put them in named or unnamed struct in a variable called receive or rcv so we reference them by rcv.packetsMissing